saleor/graphql/account/types.py

```diff
@@ -415,6 +421,21 @@ class User(ModelObjectType[models.User]):
         required=True,
         description="The data when the user last update the account information.",
     )
+    stored_payment_methods = NonNullList(
```

Comment:
What about the `stored_payment_sources` field? What's the difference between these fields?

Reply:
stored_payment_sources is dedicated to the PaymentPlugin and will be deprecated with PaymentPlugins in the future.
Also, we didn't want to use it as it would mean that we would have to have a single type that would need to work for previous solutions and the new one - it means, that new fields which are currently - `required` would need to be optional, or fulfilled with placeholder data (if even possible). 
Providing a new interface, for a new flow is much more cleaner and flexible than trying to use the old one.

Reply:
Maybe that should also be mentioned in the description, these two fields seem nearly identical for the users.